app-editors/levee/levee-4.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Really tiny vi clone, for things like rescue disks"
+HOMEPAGE="https://www.pell.portland.or.us/~orc/Code/levee/"
+SRC_URI="https://www.pell.portland.or.us/~orc/Code/levee/${P}.tar.bz2"
+
+LICENSE="levee"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+
+RDEPEND="
+	!app-text/lv
+	sys-libs/ncurses:0=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-fix-build-for-clang16.patch
+	"${FILESDIR}"/${P}-tgetent-ncurses-gentoo.patch
+)
+
+src_configure() {
+	export AC_CPP_PROG="$(tc-getCPP)"
+	export AC_PATH="${PATH}"
+	export AC_LIBDIR="$($(tc-getPKG_CONFIG) --libs curses ncurses termcap)"
+
+	./configure.sh \
```

Comment:
Could you explain the size thing?

Reply:
configure.sh of this thing expects size to be a number. So when in is set as SIZE="llvm-size" for example, it will complain that llvm-size is not a number. 
It falls back to 256000 on anthing but DOS and Atari if size is not set. So I assume setting 256000 is alright.

```
case X"${SIZE}" in
X[0-9][0-9]*)    ;;
X[0-9][0-9]*[Ll]);;
X)               ;;
X*)              AC_FAIL "--size=$SIZE is not a valid number" ;;
esac
...
if [ "$OS_DOS" ]; then
    AC_DEFINE   SIZE ${SIZE:-32000}
    AC_DEFINE   PROC    _fastcall
    AC_DEFINE   TTY_ANSI        1
    AC_CHECK_FUNCS      basename
elif [ "$OS_ATARI" ]; then
    AC_DEFINE   SIZE ${SIZE:-32000}
    AC_DEFINE   TTY_VT52        1
    AC_DEFINE   HAVE_BLKFILL    1
    AC_CHECK_FUNCS      basename
elif [ "$OS_FLEXOS" ]; then
    AC_DEFINE   SIZE ${SIZE:-256000}
    AC_CHECK_FUNCS      basename
else
    AC_DEFINE   SIZE ${SIZE:-256000}
    AC_DEFINE   OS_UNIX 1
```

Reply:
Ah! Now I get it. Could you put a comment explaining that 'size' does not mean the tool and we use the value it woudl reduce for non DOS/Atari?

Reply:
will do